### PR TITLE
Explorer Search, Path and Index functionality tip

### PR DESCRIPTION
**Pull request by jjw24**
Explorer Search, Path and Index functionality tip

### CODE REVIEW

File: usage-tips.md [hunk 1]
```diff
@@ -18,3 +18,4 @@
 - Prioritize the plugin results order, this can be done by going to the plugin's settings page. Under the plugin's title and the description, click the number next to 'Priority', this is where you can assign weight to the plugin's result. The higher the weight is, the higher the selected plugin's results will be in Flow's result list.
 - Press ctrl + enter/click on a Shell plugin command will run it directly as admin.
 - In the plugins download list, you can press ctrl + enter/click to open the plugin's url.
+- Explorer's Search action keyword combines both Path and Index search, so you can use it without worrying about which action keyword to use for what, just put in what you need to search for. Path (searches a specific path) and Index (search a file or folder name) search allows users to do just their specific searches and are disabled by default.
```
Comment (Zeroto521) [inline usage-tips.md:21]: Does put two pictures could be better?

**Review by Zeroto521 — APPROVED** [review]

File: usage-tips.md (hunk 1 above)
Comment (jjw24) [reply usage-tips.md:21]: Sorry do you mean put pictures?

Comment (Zeroto521) [reply usage-tips.md:21]: I though this has a bit long.
If there have two demo pictures to show the using could be better.

Comment (jjw24) [reply usage-tips.md:21]: oh ok. Ideally I like to break them into sections and yes add some pics/gifs. but it's something for another day lol

Comment (deefrawley) [reply usage-tips.md:21]: This whole section does need a rework - it's just a slab of dot points which isn't very user friendly. If you want, merge this change and I will have a go at reworking with better sections and some screenshots in the next week or so.

Comment (jjw24) [reply usage-tips.md:21]: Sounds good thank you, it's merged

**Pull request merged**